src/main/java/com/liferay/ide/idea/bnd/psi/util/BndPsiUtil.java

```diff
@@ -197,16 +197,18 @@ public static PsiClass getBundleActivatorClass(@NotNull PsiElement psiElement) {
 	public static PsiDirectory[] resolvePackage(@NotNull PsiElement psiElement, @NotNull String packageName) {
 		Project project = psiElement.getProject();
 
-		Module module = ModuleUtilCore.findModuleForPsiElement(psiElement);
+		//		INTELLIJ-111 Improve parsing bnd.bnd file
```

Comment:
hey @dmarks2 do you have any thoughts on this issue?  You can see the original ticket here: https://issues.liferay.com/browse/INTELLIJ-111

Reply:
Hey @simonjhy  we need to either completely remove this code if it is invalid or find a better fix.  I don't want to just comment it out to fix it.

Reply:
hey @gamerson,

I will have a look at the JIRA issue above today.

Reply:
@gamerson I found some issues on resolving the package names. The problem occurs when creating the annotation (which leads to the error message in the above ticket) as well as when resolving the reference (Ctrl-Click on the package name). However it seems that this only occurs on gradle based projects and not on maven projects. 

I will fix the issue on gradle projects and send a PR soon. 

Additionally I found a strange error when trying to rename a package. At least in a "formatted" bnd.bnd file that sometimes just removes the whole package name from the bnd.bnd file. I will have a look at this issue, too? Do you want to create a separate JIRA issue for this?